config/locales/forms.yml

```
@@ -266,6 +276,9 @@ en:
         salary_types_options:
           full_time: This is what the annual salary would be for someone working full time
           part_time: This is the annual amount someone will take home
+      publishers_job_listing_religious_information_form:
+        religion_type_options:
+          catholic: This form has been approved by the Catholic Education Service.
```

```suggestion
          catholic: This form caters for Catholic schools.
```
@starswan could we tweak this please :)